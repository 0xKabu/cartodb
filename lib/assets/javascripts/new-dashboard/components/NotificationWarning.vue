<template>
<div class="notification">
  <div v-html="htmlBody" class="notification-html text is-caption"></div>
</div>
</template>

<script>
export default {
  name: 'NotificationWarning',
  props: {
    htmlBody: String
  }
};
</script>
<style scoped lang="scss">
@import 'new-dashboard/styles/variables';

.notification {
  color: $text__color;
  padding: 16px 112px;
  background-color: $warning__bg-color;
}
</style>
