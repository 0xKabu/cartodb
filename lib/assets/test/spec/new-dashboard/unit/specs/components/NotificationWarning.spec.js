import {
  mount
} from '@vue/test-utils';
import NotificationWarning from 'new-dashboard/components/NotificationWarning';

describe('NotificationWarning.vue', () => {
  let notificationCardWrapper;

  beforeEach(() => {
    notificationCardWrapper = mount(NotificationWarning);
  });

  describe('basic behaviour', () => {
    it('should render the body', () => {
      notificationCardWrapper.setProps({
        htmlBody: 'Notification message'
      });
      expect(notificationCardWrapper).toMatchSnapshot();
    });
  });
});
