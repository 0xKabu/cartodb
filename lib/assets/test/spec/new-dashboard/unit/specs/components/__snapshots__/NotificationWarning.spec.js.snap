// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationWarning.vue basic behaviour should render the body 1`] = `
<div class="notification">
  <div class="text is-caption">Notification message</div>
</div>
`;
